game/script-farewells.rpy

```diff
@@ -859,8 +859,9 @@ label farewell_short_session_ask:
     n "What?{w=0.2} You're leaving?{w=0.2} But you've barely been here at all today,{w=0.1} [player]!"
     $ time_in_session_descriptor = utils.get_time_in_session_descriptor()
     n "In fact, you've only been here for [time_in_session_descriptor]!"
-    n "You're sure you can't stay just a little longer?"
     menu:
+        n "You're sure you can't stay just a little longer?"
```

Comment:
Keeping note of this for future when we add expressions. Lines inside the `menu` block **cannot** have expressions. Because Ren'Py.

Leaving this as a comment for if you want to keep these before or not

Reply:
_Because Ren'Py_

Jokes aside, we can at least have Nat narrating the menu choice, even if we can't supply an expression alongside it. The previously set expression should still apply, I think? So long as the expression before the menu fits the question Nat is about to ask (effectively always), I think we're okay.

If not - we'll have to revisit all dialogue anyway once sprites are in; a few menu lines aren't much more to deal with.